js/tests/unit/modal.spec.js

```diff
@@ -62,170 +61,22 @@ describe('Modal', () => {
     it('should toggle a modal', done => {
       fixtureEl.innerHTML = '<div class="modal"><div class="modal-dialog"></div></div>'
 
-      document.documentElement.style.overflowY = 'scroll'
+      const initialOverFlow = document.body.style.overflow
       const modalEl = fixtureEl.querySelector('.modal')
       const modal = new Modal(modalEl)
-      const originalPadding = '0px'
+      const originalPadding = '10px'
 
       document.body.style.paddingRight = originalPadding
 
       modalEl.addEventListener('shown.bs.modal', () => {
         expect(document.body.getAttribute('data-bs-padding-right')).toEqual(originalPadding, 'original body padding should be stored in data-bs-padding-right')
```

Comment:
Should this be moved to scrollbar spec as well, as the data attribute is set by the scrollbar script?

Reply:
As a matter of fact yes. 
The grater purpose is to refactor the scrollbar.js to class, and to support different root elements.
So I left on test to **spyOn** on the next PR trying to minimize the changes.

The original checks/test are done on scrollbar.spec